.github/workflows/ci.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         node-version:
           - 14.x
```

Comment:
Since Node 18 is LTS now, could you remove 14 and 16 everywhere, and add 18 and 20 instead? (if only a single version is mentioned, it can be 18)